customer-service/omnichannel/real-time-translation/README.md

```diff
@@ -11,15 +11,15 @@ The real time translation behavior is defined by the web resource, therefore, ve
 
 The webResourceV2.js is a sample web resource that you can use to enable real-time translation of the conversation messages exchanged between the customer and the agent. Perform the following steps before using the sample:
 
-1. Directly download and change the API key for the translation API engine, being used in your organisation, in `webResourceV2.js` at:
+1. Directly download the file and change the API key for the translation API engine that is used in your organisation, in `webResourceV2.js` at:
 
 ```
 	bingTranslateApiClientSecret: '<please add your own azure translation api key>',
 	googleTranslateApiClientSecret: '<please add your own google translation v2 api key>',
 	useAzureTranslationApis: true,//please override it to false if planning to use google translation v2 api
 ```
 
-2. Additional changes can be done in the web resource logic as per your business needs.
+2. Make additional changes in the web resource logic as per your business needs.
```

Comment:
this is optional step. So, please either revert it or change it such that it looks like an optional step